packages/utils/functions/token/getToken/index.ts

```diff
@@ -1,10 +1,10 @@
 import ClientCookies from 'js-cookie'
-import { cookies as serverCookies } from 'next/headers'
```

Comment:
This function was giving me some issues so I opted out from using it in the project that needed it. It looks like nextjs doesn't like us to use things from next/headers outside the server side. Maybe getting the token in the server vs the client should be separate functions in their own files. Don't know if using a dynamic import fixes it, I didn't test it out...